jwst/outlier_detection/utils.py

```diff
@@ -143,6 +250,31 @@ def flag_crs_in_models_with_resampling(
         # dq flags will be updated in-place
         flag_resampled_model_crs(image, blot, snr1, snr2, scale1, scale2, backg)
 
+def flag_crs_in_models_with_resampling_library(
```

Comment:
can this just be replaced with a call to `map_function()` like for flag_crs_in_models_library()?